_rules/no-auto-play-audio-80f0bf.md

```diff
@@ -0,0 +1,145 @@
+---
+id: 80f0bf
+name: Video or audio has no auto-play audio.
+rule_type: composite
+description: |
+    This rule checks that auto-play audio does not last for more than 3 seconds, or the audio has a control mechanism to stop or mute the it.
+accessibility_requirements:
+  wcag20:1.4.2: # Audio Control (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - 95e043
+  - aaa1bf
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn ANderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds, that has an `autoplay` attribute equal to `true`, and that has both `paused` and `muted` attributes equal to `false`.
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+- [Auto-play audio does not exceed 3 seconds](https://act-rules.github.io/rules/auto-play-audio-has-control-mechanism-95e043.md)
+- [Auto-play audio has control mechanism](https://act-rules.github.io/rules/auto-play-audio-exceeds-3-seconds-aaa1bf.md)
```

Comment:
the actual file is named `_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md`, this link needs to be updated.